app/controllers/additional_document_validation_requests_controller.rb

```diff
@@ -58,4 +61,12 @@ def additional_document_validation_request_params
   def set_additional_document_validation_request
     @additional_document_validation_request = @planning_application.additional_document_validation_requests.find(params[:id])
   end
+
+  def cancel_redirect_url
```

Comment:
Do we have a method for this already or is this redirect behaviour supposed to be different for this type of request?

https://github.com/unboxed/bops/blob/main/app/controllers/concerns/validation_requests.rb#L105

Reply:
I'm deliberately over-riding that method here because it takes the user to the post-validation requests table. Since they navigate to cancelling the document via the documents page, I think it's maybe bit disorientating to take them to the table page - they won't necessarily have every been there before 😅 .

Reply:
Ah okay sounds good. We may wanna speak with Tom about some consistency here going forward . I reckon after we finish these PR's for post validation requests we'll do a cleanup of some bits